app/controllers/api/cars_controller.rb

```diff
@@ -33,6 +33,14 @@ def update
       end
     end
 
+    def destroy
+      @car = Car.find(params[:id])
+      if @car.destroy
+        render json: { reservation: @reservation, message: 'Reservation successfully deleted' }

```

Comment:
You should handle the success of the destroying car success in the same as you had handled the not success . Then, yo should set status: :ok for this successful case because I'm controlling this response in the front end.